src/components/SearchBar.tsx

```diff
@@ -57,18 +57,30 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     clearInput() {
         let searchInput: any = document.getElementById("SearchInputText");
         searchInput.value = '';
-        this.setState({ hasText: false });
+        this.setState({
+            hasText: false,
+            expanded: false // collapse filter options menu when text is cleared
+        });
```

Comment:
I'm thinking out loud... but it seems like we can just keep `hasText` and remove `expanded`, if we don't allow expansion when there is no search text. Can this be optimized/streamlined?